content/ending-frozen-applications.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Ending Frozen Applications and Desktops
+description: >
+  Linux provides several methods to kill frozen or hung up processes and applications using GUI applications or terminal commands.
+keywords:
+  - frozen applications
+  - frozen desktop
+  - frozen terminal
+  - hung up
+  - not responding
+  - System76
+
+facebookImage: /_social/article
+twitterImage: /_social/article
+
+hidden: false
+section:
+---
+
+## Ending Frozen Processes, Applications, and Desktops
+
+Linux offers many methods to end hanging or frozen processes. The `gnome-system-monitor` provides a GUI process management application with features similar to the Windows Task Manager, while the `kill` command can end single or multiple processes from the Terminal.
+
+>Note: This article details several methods to end processes for a given scenario, but is not exhaustive or guaranteed to provide the most appropriate method for every scenario. Users should also keep in mind that suddenly terminating processes can result in unwanted data loss or system stability issues.
+
+## End Specific Applications or Windows
+
+### Using gnome-system-monitor (GUI)
+
+The gnome-system-monitor comes prepackaged in Pop!_OS. This application provides a GUI interface and functionality similar to the Windows Task Manager.
+
+1. Press <kbd>Super</kbd> + <kbd>T</kbd> to launch a Terminal session.
+2. Enter `gnome-system-monitor` and hit <kbd>Enter</kbd>.
+3. Locate frozen and hanging applications.
+
```

Comment:
This entire list is being rendered with excessive space between the list items due to these empty lines; leave the empty lines out to keep the list together. Compare how it looks now:
![Screenshot from 2022-02-09 11-34-14](https://user-images.githubusercontent.com/7199422/153267168-b06649f6-408e-4e00-9f6b-8fa89dbcdc10.png)
With how it should look:
![Screenshot from 2022-02-09 11-33-57](https://user-images.githubusercontent.com/7199422/153267166-a52731eb-bc0e-4280-92c9-b40e576d15fc.png)

Reply:
I tried to get npm installed with Nathaniel but it kept giving some kind of error. Is there some way I can get this preview before making a pull request?

Reply:
I think I fixed the spacing, let me know if not

Reply:
![image](https://user-images.githubusercontent.com/80931038/153285580-bccf4bfd-e3fc-447f-a8df-dc7165917de7.png)
removing those lines causes the linter to get angry at me, maybe I'm not understanding your comment

Reply:
That check in the linter is clearly not reasonable, since doing what it suggests changes the displayed formatting on the page. Let me see the best way to fix/work around that.

Reply:
I disabled the MD031 check, the lint is now passing.